tests/test_transport.py

```diff
@@ -196,9 +196,9 @@ def setup_test_server(
     def test_security_options(self):
         o = self.tc.get_security_options()
         self.assertEqual(type(o), SecurityOptions)
-        self.assertTrue(("aes256-cbc", "blowfish-cbc") != o.ciphers)
-        o.ciphers = ("aes256-cbc", "blowfish-cbc")
-        self.assertEqual(("aes256-cbc", "blowfish-cbc"), o.ciphers)
+        self.assertTrue(("aes256-cbc", "aes192-cbc") != o.ciphers)
```

Comment:
For my info -- why the change from `blowfish-cbc` to `aes192-cbc` here? Would it make sense to add additional ciphers to this test?

Reply:
I just changed it to another valid cipher that was in the list to maintain the functionality of the test where it validates the cipher list can be overridden.  I just picked one at random effectively

Reply:
Gotcha, figured that was it. Thanks!